libexec/rc/rc.conf

```diff
@@ -121,9 +122,9 @@ hostid_file="/etc/hostid"	# File with hostuuid.
 nisdomainname="NO"		# Set to NIS domain if using NIS (or NO).
 dhclient_program="/sbin/dhclient"	# Path to dhcp client program.
 dhclient_flags=""		# Extra flags to pass to dhcp client.
-#dhclient_flags_em0=""		# Extra dhclient flags for em0 only
+#dhclient_flags_fxp0=""		# Extra dhclient flags for fxp0 only
```

Comment:
Cosmetic change that shows the age of the original file copy. Might want to remove this change to make it easier to resync with upstream file later on.